vector/src/main/res/values-fr/strings.xml

```diff
@@ -874,6 +874,7 @@ Appareils inconnus :</string>
     <string name="tchap_register_unauthorized_email">Cette adresse e-mail n\'est pas autorisée</string>
     <string name="tchap_register_wait_for_email_caution">Veuillez rester sur cet écran jusqu\'à la fin de la procédure d\’inscription</string>
     <string name="tchap_register_wait_for_email_prompt_on_back">Votre inscription n\’est pas terminée. Etes-vous sûr de vouloir abandonner la procédure d\’inscription ?</string>
+    <string name="tchap_auth_agent_failure_warning_msg">La connexion au serveur agent a échoué. Seules les données sécurisées seront disponibles.\nVous pourrez tenter une nouvelle connexion sur ce serveur agent depuis les paramètres.</string>
```

Comment:
This PR does not include change in the settings. Is it a future feature?

Reply:
Yes this option will be added later in the settings

Reply:
ok